From 43df8b9e7a7032e467742a7a75f23c54564dbfa4 Mon Sep 17 00:00:00 2001
Message-Id: <43df8b9e7a7032e467742a7a75f23c54564dbfa4.1684168831.git.stefan@agner.ch>
In-Reply-To: <c46f6cb84fb189da723a9932a5195a553762e2b6.1684168831.git.stefan@agner.ch>
References: <c46f6cb84fb189da723a9932a5195a553762e2b6.1684168831.git.stefan@agner.ch>
From: Stefan Agner <stefan@agner.ch>
Date: Mon, 19 Sep 2022 23:05:51 +0200
Subject: [PATCH] Revert Python dependencies

Remove the crypto libraries. They are not required in regular use.
---
 src/controller/python/BUILD.gn | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/controller/python/BUILD.gn b/src/controller/python/BUILD.gn
index 424ed433b1..2d7ff2de63 100644
--- a/src/controller/python/BUILD.gn
+++ b/src/controller/python/BUILD.gn
@@ -318,11 +318,6 @@ chip_python_wheel_action("chip-core") {
     "ipdb",
     "deprecation",
     "mobly",
-
-    # Crypto libraries for complex tests and internal Python controller usage
-    "cryptography",
-    "pycrypto",
-    "ecdsa",
   ]
 
   if (current_cpu == "x64") {
-- 
2.40.1

